tests/test_autoray.py

```diff
@@ -545,3 +545,31 @@ def test_einsum(backend):
         == ar.infer_backend(C4)
         == backend
     )
+
+
+@pytest.mark.parametrize("backend", BACKENDS)
+def test_split(backend):
+    if backend == "sparse":
+        pytest.xfail("sparse doesn't support split yet")
+    if backend == "dask":
+        pytest.xfail("dask doesn't support split yet")
+    A = ar.do("ones", (10, 20, 10), like=backend)
+    sections = [2, 4, 14]
```

Comment:
Is it possible to parametrize ``sections`` with a int case as well? then we'll have full test coverage for the translations.

Reply:
Good thing you noticed, implementation for int case for torch was wrong (torch takes split _size_ as opposed to number of splits), should be fixed now.